infra/azure/k8s/cilium/argo_server.yaml

```
@@ -0,0 +1,53 @@
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  name: argo-workflows-server-cnp
```

```suggestion
  name: argo-workflows-server
```

Probably don't need cnp here as the object is of type cnp.